### DISCUSSION TITLE
Some events missing from alliedmods

### DISCUSSION THREAD

**Discussion by zoidyzoidzoid** [saul/demofile#225] [Questions] [answered]
I found the following events in my demo that aren't in AlliedMods. Can't figure out how to make an account there though. 🤔 

```
begin_new_match
cs_round_final_beep
cs_round_start_beep
hltv_chase
hltv_status
item_remove
player_disconnect
player_spawn
player_team
round_announce_last_round_half
round_announce_match_point
round_announce_match_start
round_officially_ended
round_time_warning
```

P.S. Thanks for the awesome library.

**Comment by zoidyzoidzoid** [comment]
I figured it out, I just needed to create a forums account [here](https://forums.alliedmods.net/register.php)

**Comment by saul** [comment, marked as answer]
I believe they’re on the wiki here:

https://wiki.alliedmods.net/Generic_Source_Events
https://wiki.alliedmods.net/Generic_Source_Server_Events

**Comment by zoidyzoidzoid** [comment]
Oh, damn, you're right. A bunch of them are. I'll remove those ones from the CS:GO wiki page.
